### PR TITLE
Change the links in Chinese docs

### PR DESCRIPTION
**Pull request by LuQQiu**
Change the relative urls in Chinese docs

### CODE REVIEW

File: docs/cn/compute/Hadoop-MapReduce.md [hunk 1]
```diff
@@ -14,16 +14,16 @@ priority: 1
...
+- 你已经按照这些文档建立了一个Alluxio集群: [本地模式]({{ '/cn/deploy/Running-Alluxio-Locally.html' | relativize_url }})或[集群模式]({{ '/cn/deploy/Running-Alluxio-on-a-Cluster.html' | relativize_url }})。
 - 为了运行一些简单的map-reduce实例，我们也推荐你下载根据你的hadoop版本对应的[map-reduce examples jar](http://mvnrepository.com/artifact/org.apache.hadoop/hadoop-mapreduce-examples/2.4.1)，或者如果你正在使用Hadoop 1，下载这个[examples jar](http://mvnrepository.com/artifact/org.apache.hadoop/hadoop-examples/1.2.1)。
 
 ## 编译Alluxio客户端
 
 为了使MapReduce应用可以与Alluxio进行通信，你需要将Alluxio Client的Jar包包含在MapReduce的classpaths中。我们建议你从Alluxio [download page](http://www.alluxio.io/download) 下载压缩包。
 
 同时，高级用户可以选择使用源代码来编译生成Alluxio Client的Jar包。
-你可以运行以下命令[here](Building-Alluxio-From-Source.html#compute-framework-support)。
+你可以运行以下命令[here]({{ '/cn/contributor/Building-Alluxio-From-Source.html' | relativize_url }}#compute-framework-support)。
```
Comment (jiacheliu3) [inline docs/cn/compute/Hadoop-MapReduce.md]: here -> 编译Alluxio源代码?

File: docs/cn/contributor/Contributor-Getting-Started.md [hunk 2]
```diff
@@ -224,11 +224,11 @@ $ git push origin awesome_feature
...
 新开发者可以通过以下资料和教程来进一步熟悉并贡献Alluxio:
 
-1. [本地运行Alluxio](Running-Alluxio-Locally.html)
-1. [集群上运行Alluxio](Running-Alluxio-on-a-Cluster.html)
+1. [本地运行Alluxio]({{ '/cn/deploy/Running-Alluxio-Locally.html' | relativize_url }})
+1. [集群上运行Alluxio]({{ '/cn/deploy/Running-Alluxio-on-a-Cluster.html.html' | relativize_url }})
 1. 阅读[配置文档]({{ '/cn/operation/Configuration.html' | relativize_url }})和[命令行相关操作]({{ '/cn/operation/User-CLI.html' | relativize_url }})
 1. 阅读[示范代码](https://github.com/alluxio/alluxio/blob/master/examples/src/main/java/alluxio/examples/BasicOperations.java)
-1. [从源码编译Alluxio](Building-Alluxio-From-Source.html)
+1. [从源码编译Alluxio]({{ '/cn/contributor/Building-Alluxio-From-Source.html' | relativize_url }})
```
Comment (jiacheliu3) [inline docs/cn/contributor/Contributor-Getting-Started.md]: 从源码编译Alluxio vs 编译Alluxio源代码？

File: docs/cn/deploy/Running-Alluxio-Locally.md [hunk 3]
```diff
@@ -46,7 +46,7 @@ $ ./bin/alluxio-start.sh local
 
 > 注意：用户在linux系统下运行上述命令需要输入密码来获取sudo权限,
 > 以便启动RAMFS。如果用户不想每次运行命令输入密码，
-> 或者没有sudo权限，可以使用[FAQ](#faq)中介绍的其他方法。
+> 或者没有sudo权限，可以使用[FAQ]({{ '/cn/overview/FAQ.html' | relativize_url }})中介绍的其他方法。
```
Comment (jiacheliu3) [inline docs/cn/deploy/Running-Alluxio-Locally.md]: FAQ -> 常见问题？

File: docs/cn/deploy/Running-Alluxio-On-Kubernetes.md [hunk 4]
```diff
@@ -16,7 +16,7 @@ Alluxio可以在Kubernetes上运行。这个指南演示了如何使用Alluxio G
 
 - 一个Kubernetes集群 (版本 >= 1.8). Alluxio workers将使用`sizeLimit`参数来限制使用`emptyDir`卷的大小。这是Kubernetes 1.8的一个alpha特性。
 请确保该功能已启用。
-- 一个Alluxio Docker镜像。请参阅[本页](Running-Alluxio-On-Docker.html)了解如何构建镜像。该镜像必须可以从运行Alluxio进程的所有Kubernetes主机提取。这可以通过将镜像推送到可访问的Docker注册表来实现，或者将镜像单独推送到所有主机。如果使用私人Docker注册表，请参阅Kubernetes [文档](https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/)。
+- 一个Alluxio Docker镜像。请参阅[本页]({{ '/cn/deploy/Running-Alluxio-On-Docker.html' | relativize_url }})了解如何构建镜像。该镜像必须可以从运行Alluxio进程的所有Kubernetes主机提取。这可以通过将镜像推送到可访问的Docker注册表来实现，或者将镜像单独推送到所有主机。如果使用私人Docker注册表，请参阅Kubernetes [文档](https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/)。
```
Comment (jiacheliu3) [inline docs/cn/deploy/Running-Alluxio-On-Kubernetes.md]: 本页 -> 用Docker运行Alluxio？

File: docs/cn/deploy/Running-Alluxio-On-Kubernetes.md [hunk 5]
```diff
@@ -63,7 +63,7 @@ $ kubectl create -f alluxio-journal-volume.yaml
 ```
 
 ## 配置Alluxio属性
-Kubernetes中的Alluxio容器使用环境变量来设置Alluxio属性。有关`conf/alluxio-site.properties`中的Alluxio属性的相应环境变量名称，请参阅[Docker配置](Running-Alluxio-On-Docker.html)。
+Kubernetes中的Alluxio容器使用环境变量来设置Alluxio属性。有关`conf/alluxio-site.properties`中的Alluxio属性的相应环境变量名称，请参阅[Docker配置]({{ '/cn/deploy/Running-Alluxio-On-Docker.html' | relativize_url }})。
```
Comment (jiacheliu3) [inline docs/cn/deploy/Running-Alluxio-On-Kubernetes.md]: Docker配置 -> 用Docker运行Alluxio

File: docs/cn/deploy/Running-Alluxio-on-a-Cluster.md [hunk 6]
```diff
@@ -29,7 +29,7 @@ $ cp conf/alluxio-site.properties.template conf/alluxio-site.properties
 
 更新`conf/alluxio-site.properties`中的`alluxio.master.hostname`为你将运行Alluxio Master的机器的主机名。添加所有worker节点的IP地址到`conf/workers`文件。
 如果集群中存在多节点，你不可以使用本地文件系统作为Allxuio底层存储层。你需要在所有Alluxio服务端连接的节点启动共享存储，共享存储可以是
-网络文件系统（NFS），HDFS，S3等。例如。你可以参照[Configuring Alluxio with S3](Configuring-Alluxio-with-S3.html)按照说明启动S3作为Alluxio底层存储。
+网络文件系统（NFS），HDFS，S3等。例如。你可以参照[Configuring Alluxio with S3]({{ '/cn/ufs/S3.html' | relativize_url }})按照说明启动S3作为Alluxio底层存储。
```
Comment (jiacheliu3) [inline docs/cn/deploy/Running-Alluxio-on-a-Cluster.md]: Configuring Alluxio with S3 -> 配置Alluxio使用S3？

File: docs/cn/operation/Journal.md [hunk 7]
```diff
@@ -56,7 +56,7 @@ $ ./bin/alluxio fsadmin backup
 alluxio.master.backup.directory=/alluxio/backups
 ```
 
-查看 [backup command documentation](Admin-CLI.html#backup) 获取写备份文件具体位置的额外配置。
+查看 [backup command documentation]({{ '/cn/operation/Admin-CLI.html' | relativize_url }}#backup 获取写备份文件具体位置的额外配置。
```
Comment (jiacheliu3) [inline docs/cn/operation/Journal.md]: backup command documentation -> backup指令

File: docs/cn/operation/Server-Logging.md [hunk 8]
```diff
@@ -114,7 +114,7 @@ log4j.appender.USER_LOGGER.layout.ConversionPattern=%d{ISO8601} %-5p %c{1} - %m%
 Alluxio支持通过网络向远程的日志服务器发送日志。这个特性对于必须完成日志收集任务的系统管理者来说十分有用。得益于远程日志汇总功能，所有Alluxio服务器上的日志文件，例如master.log, worker
 .log等，可以在日志服务器上的指定路径轻松获取，且该路径可配置。
 
-如要获取在集群上部署Alluxio的指导，请参考 [Running Alluxio on a cluster](Running-Alluxio-on-a-Cluster.html)
+如要获取在集群上部署Alluxio的指导，请参考 [Running Alluxio on a cluster]({{ '/en/deploy/Running-Alluxio-on-a-Cluster.html' | relativize_url }})
```
Comment (jiacheliu3) [inline docs/cn/operation/Server-Logging.md]: Running Alluxio on a cluster -> 集群运行Alluxio?

File: docs/cn/operation/Troubleshooting.md [hunk 9]
```diff
@@ -125,9 +125,9 @@ $ ls {{site.ALLUXIO_CLIENT_JAR_PATH}}
 - 请确保Alluxio的master节点和client节点的安全设置保持一致.
 Alluxio通过配置`alluxio.security.authentication.type`来提供不同的[用户身份验证]({{ '/cn/operation/Security.html' | relativize_url }}#authentication)的方法。
 如果客户端和服务器的这项配置属性不一致，这种错误将会发生。(例如，客户端的属性为默认值`NOSASL`,而服务器端设为`SIMPLE`)
-有关如何设定Alluxio的集群和应用的问题，用户请参照[Configuration-Settings]({{ '/cn/operation/Configuration.html' | relativize_url }})
+有关如何设定Alluxio的集群和应用的问题，用户请参照[配置文档]({{ '/cn/operation/Configuration.html' | relativize_url }})
 - Spark调用Alluxio-1.3.0文件时报错，如果是直接下载编译好的alluxio文件进行安装的，一般会出现该错误。
-解决办法：需要Alluxio client需要在编译时指定Spark选项，具体参考[Running-Spark-on-Alluxio](Running-Spark-on-Alluxio.html)；
+解决办法：需要Alluxio client需要在编译时指定Spark选项，具体参考[Running-Spark-on-Alluxio]({{ '/cn/compute/Spark.html' | relativize_url }})；
```
Comment (jiacheliu3) [inline docs/cn/operation/Troubleshooting.md]: Running-Spark-on-Alluxio -> 在Alluxio上运行Spark？

File: docs/cn/overview/FAQ.md [hunk 10]
```diff
@@ -27,16 +27,16 @@ priority: 10
 
 5. Alluxio master节点是否具备容错能力？
 
-   是的。请参考[此页](Running-Alluxio-on-a-Cluster.html#running-alluxio-with-high-availability)。
+   是的。请参考[此页]({{ '/cn/deploy/Running-Alluxio-on-a-Cluster.html' | relativize_url }}#running-alluxio-with-high-availability)。
```
Comment (jiacheliu3) [inline docs/cn/overview/FAQ.md]: 此页 -> 集群运行Alluxio？

File: docs/cn/overview/FAQ.md [hunk 11]
```diff
@@ -49,20 +49,20 @@ priority: 10
 
 10. Alluxio可以作为一个单机系统运行以进行实验和验证吗？
 
-    是的，Alluxio可以作为一个单机系统运行。[这里](Running-Alluxio-Locally.html)是操作指南。
+    是的，Alluxio可以作为一个单机系统运行。[这里]({{ '/cn/deploy/Running-Alluxio-Locally.html' | relativize_url }})是操作指南。
```
Comment (jiacheliu3) [inline docs/cn/overview/FAQ.md]: 这里 -> 本地运行Alluxio？

File: docs/cn/overview/FAQ.md [hunk 12]
```diff
@@ -49,20 +49,20 @@ priority: 10
...
 
 11. Alluxio可以以集群模式运行吗？
 
-    是的，Alluxio可以以集群模式运行。[这里](Running-Alluxio-on-a-Cluster.html)是操作指南。
+    是的，Alluxio可以以集群模式运行。[这里]({{ '/cn/deploy/Running-Alluxio-on-a-Cluster.html' | relativize_url }})是操作指南。
```
Comment (jiacheliu3) [inline docs/cn/overview/FAQ.md]: 这里 -> 本地运行Alluxio？

File: docs/en/overview/FAQ.md [hunk 13]
```diff
@@ -65,7 +65,7 @@ Read more about Alluxio storage setup
 ## Does Alluxio support a high availability mode?
 
 Yes. See instructions about
-[Deploy Alluxio on a Cluster with HA]({{ '/en/install/Running-Alluxio-On-a-HA-Cluster.html' | relativize_url }}).
+[Deploy Alluxio on a Cluster with HA]({{ '/en/deploy/Running-Alluxio-On-a-HA-Cluster.html' | relativize_url }}).
```
Comment (jiacheliu3) [inline docs/en/overview/FAQ.md]: Deploy Alluxio on a Cluster with HA -> 部署高可用Alluxio集群？

**Comment by jiacheliu3** [comment]
wait @Xenorith I requested changes

Never expected to be this much to review too LOL

File: docs/en/overview/FAQ.md (hunk 13 above)
Comment (LuQQiu) [reply docs/en/overview/FAQ.md]: hahaha, this is an leftover English doc

**Comment by LuQQiu** [comment]
@jiacheliu3 hahaha thanks for the really detailed review!! Fixed all, PTAL

**Comment by LuQQiu** [comment]
alluxio-bot, merge this please

**Pull request merged**